02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/03 Scikit-Learn/08 Load Models.html

```diff
@@ -0,0 +1,7 @@
+<p>You can load and trade with pre-trained <code>sklearn</code> models that you saved in the ObjectStore. To load a <code>sklearn</code> model from the ObjectStore, in the <code>Initialize</code> method, get the file path to the saved model and then call the <code>load</code> method.</p>
+<div class="section-example-container">
+    <pre class="python">def Initialize(self) -&gt; None:
+    if self.ObjectStore.ContainsKey(model_key):
+        file_name = self.ObjectStore.GetFilePath(model_key)
+        self.model = joblib.load(file_name)</pre>
+</div>
```

Comment:
After this line add the text from the keras tutorial that says "The ContainsKey method returns ..."